src/service/cid-impl.js

```diff
@@ -305,9 +305,10 @@ export class Cid {
           if (scope) {
             apiKeyMap[scope] = API_KEYS[clientName];
           } else {
-            user().error(
+            user().warn(
               TAG_,
-              `Unsupported client for Google CID API: ${clientName}`
+              `Unsupported client for Google CID API: ${clientName}. 
+Please remove or correct ${optInMeta./*OK*/ outerHTML}`
```

Comment:
Drive-by review: If you want to avoid the `\n` in the warning string, you can rewrite this as:
```js
            user().warn(
              TAG_,
              `Unsupported client for Google CID API: ${clientName}. ` +
                `Please remove or correct ${optInMeta./*OK*/ outerHTML}`
            );
```

Reply:
sure, updated